### PR TITLE
added custom_field documentation and configs

### PR DESCRIPTION
**Pull request by Justbkuz**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [ √] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [√ ] Update README with any necessary configuration snippets

- [√ ] Binstubs are created if needed

- [√ ] RuboCop passes

- [√ ] Existing tests pass 

#### Purpose
Add the ability to add custom fields to the messages that are sent to slack. fields added that don't exist or have no value are ignored.

image

### CODE REVIEW

**Comment by majormoses** [comment]
@Justbkuz thanks for the contribution I will take a look when I have some time.

**Comment by majormoses** [comment]
taking a quick look the only thing I can think of is the potential to leak secrets maybe we reject any values that match the sensu redact config?

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Added
 - ruby 2.4 testing in travis (@majormoses)
 
+### Added
```
Comment (majormoses) [inline CHANGELOG.md]: You can get rid of this and just leave the line below under the existing `### Added` tag.

**Comment by Justbkuz** [comment]
removed the CHANGELOG.md as suggested

**Comment by majormoses** [comment]
Sorry maybe I was not clear enough, just put your addition underneath the existing one like this:
```
## [Unreleased]
￼### Added
- ruby 2.4 testing in travis (@majormoses)
- `slack-handler-multichannel.rb`: Add custom_field options to supply additional fields (@justbkuz)

```

**Comment by Justbkuz** [comment]
Adjusted

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-slack/versions/1.2.0

**Pull request merged**